components/auth/SelectPrincipal.vue

```diff
@@ -146,6 +151,19 @@ export default {
     @input="add"
     @search="onSearch"
   >
+    <template v-slot:no-options="{ search: searchVal , searching }">
+      <template v-if="searching">
+        <span class="search-slot">
+          No results found for <em>{{ searchVal }}</em>
```

Comment:
Needs localising, also we probably don't need to show the search term as it's the line directly above.